common/services/prismic/events.js

```diff
@@ -0,0 +1,18 @@
+import {getDocument} from './api';
+import {parseBackgroundTexture} from './parsers';
+import type {UiEventSeries} from '../../model/events';
+
+export async function getUiEventSeries(req: Request, id: string): Promise<UiEventSeries> {
+  const series = await getDocument(req, id, {
+    fetchLinks: ['background-textures.image', 'background-textures.name']
+  });
+
+  const prismicBackgroundTexture = series && series.data && series.data.backgroundTexture && series.data.backgroundTexture.data;
```

Comment:
When would you ever have series and not series .data as it's part of `PrismicDocument`?

Reply:
Good point – I wasn't using `getDocument` before the move to common, and wasn't checking for a `PrismicDocument` type. I'll update.